src/EtherFiNode.sol

```diff
@@ -634,33 +626,9 @@ contract EtherFiNode is IEtherFiNode, IERC1271 {
     function _queueEigenpodFullWithdrawal() private returns (bytes32[] memory fullWithdrawalRoots) {
         if (!isRestakingEnabled) return fullWithdrawalRoots;
 
-        if (!IEigenPod(eigenPod).hasRestaked()) {
-            // EigenPod has never re-staked. Then, just withdraw the funds to the withdrawal safe
-            IEigenPod(eigenPod).withdrawBeforeRestaking();
-        } else {
-            // There are three flows of withdrawals from EL: {NonBeaconChainEthWithdrawal, partial withdraw, full withdrawal}
-            // All flows are subject to the DelayedWithdrawal put by the EL's DelayedWithdrawalRouter
-            // - In the NonBeaconChainEthWithdrawal, the verification is not required and the withdrawal is queued upon the call to `withdrawNonBeaconChainETHBalanceWei`
-            // - In the partial withdrawal, the verified withdrawal is immeidately queued 
-            // https://github.com/Layr-Labs/eigenlayer-contracts/tree/90a0f6aee79b4a38e1b63b32f9627f21b1162fbb/src/contracts/pods/EigenPod.sol#L717
-            // - In the full withdrawal, the verified withdrawal amount is kept in the EigenPod until we call `DelegationManager.queueWithdrawals`
-            // https://github.com/Layr-Labs/eigenlayer-contracts/tree/90a0f6aee79b4a38e1b63b32f9627f21b1162fbb/src/contracts/pods/EigenPod.sol#L685
-            // 
-            // Therefore, here we only need to queue {NonBeaconChainEthWithdrawal, full withdrawal}
-            _queueNonBeaconChainEthWithdrawal();
-            fullWithdrawalRoots = _queueRestakedFullWithdrawal();
-        }
-
-    }
-
-    function _queueNonBeaconChainEthWithdrawal() internal {
```

Comment:
don't we need this anymore?

Reply:
both `nonBeaconChainETHBalanceWei()` and `withdrawNonBeaconChainETHBalanceWEI` are removed in PEPE. 

All eth is treated equal in the new system even if someone cheated to send eth to the pod (block rewards, self-destruct, etc.). Unlike M2 which didn't award shares for nonBeaconEth, PEPE does reward shares for nonBeaconEth

In fact in the new system, if you don't care about actual validator rewards, the cheapest way to restake with eigenlayer is to just self-destruct a bunch of ETH into the pod :)